pkg/crds/dashboard/crds.go

```diff
@@ -125,6 +126,12 @@ func List(cfg *rest.Config) (_ []crd.CRD, err error) {
 	if features.ProvisioningV2.Enabled() {
 		result = append(result, provisioningv2.List()...)
 	}
+	for i := len(result) - 1; i >= 0; i-- {
```

Comment:
Nit: I don't think that this is the right place for this logic to go. This function just produces a list of CRDs - I think it makes more sense to have the `Create` function (which actually creates the CRDs) filter for migrated resources, in case another consuming area (I didn't see one inside of rancher, but another application could technically be using it) is using this for other reasons.

Reply:
I don't think I quite agree. I think `Create` is a function designed for creating our CRDs the way Rancher sees fit - by calling it you are accepting that Rancher may choose to install/not install whatever it wants. List, on the other hand, seems non-transformative to me, you are just asking for the list of CRDs.

Reply:
> I think Create is a function designed for creating our CRDs the way Rancher sees fit

Correct, and the current way it determines what Rancher sees fit is by calling the `List` function.

> List, on the other hand, seems non-transformative to me

`List` selectively transforms the results returned based on Rancher's state currently, we are just adding more state information (i.e. What CRDs should be defined dynamically). 
I will also note that `List` does not return all CRDs that are used by Rancher since a handful of them are installed in other locations by Norman. So the behavior of selectively returning CRDs to be installed by `Create` and not those installed by other functions is already being done.

> I also think it's worth pointing out that by having this in List you are changing the behavior of two exported functions (Create and List since Create depends on List), whereas putting it in Create only changes one. So this method better preserves backward compatibility.

This is more of a reason to change `List` because currently, you can determine what CRD will be installed by `Create` by calling `List`. Changing `Create` would cause the two functions to diverge.

Reply:
Resolving, create doesn't do stateful mutation today, but List does.